frontend/public/locales/en/translation.json

```diff
@@ -158,6 +158,7 @@
     "Automations": "Automations",
     "Autoscaling": "Autoscaling",
     "Available cluster types": "Available cluster types",
+    "Azure Resource Groups are logical collections of virtual machines, storage accounts, virtual networks, web apps, databases, and/or database servers. Typically, users group related resources for an application, divided into groups for production and non-production.": "Azure Resource Groups are logical collections of virtual machines, storage accounts, virtual networks, web apps, databases, and/or database servers. Typically, users group related resources for an application, divided into groups for production and non-production.",
```

Comment:
_Typically, users group related resources for an application_

What is this trying to say?

Reply:
This is not related to Submariner, I am not sure about the context either.

Reply:
OK, either way the sentences don't make sense so give me a minute and I can provide a suggestion to improve them.